src/Microsoft.AspNetCore.HttpOverrides/IPNetwork.cs

```diff
@@ -1,12 +1,245 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using System.Globalization;
 using System.Net;
+using System.Net.Sockets;
+using System.Collections.Immutable;
 
 namespace Microsoft.AspNetCore.HttpOverrides
 {
     public class IPNetwork
     {
+        private static readonly char[] PrefixLengthSeparators = new char[] { '/' };
+
+        /// <summary>
+        /// All IPv4 addresses 0.0.0.0/0
+        /// </summary>
+        public static readonly IPNetwork AllIPv4 = Parse("0.0.0.0/0");
+
+        /// <summary>
+        /// IPv4 localhost range 127.0.0.0/8 (255.0.0.0)
+        /// </summary>
+        public static readonly IPNetwork IPv4Loopback = Parse("127.0.0.1/8");
```

Comment:
And update the default value of ForwardedHeadersOptions.KnownNetworks to use this static